### PR TITLE
Fixing README for display on Cordova website

### PR DESCRIPTION
**Pull request by riknoll**
@dblotsky @rakatyal to review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -415,7 +411,7 @@ Starts recording an audio file.
 
 - Windows devices can use MP3, M4A and WMA formats for recorded audio. However in most cases it is not possible to use MP3 for audio recording on _Windows Phone 8.1_ devices, because an MP3 encoder is [not shipped with Windows Phone](https://msdn.microsoft.com/en-us/library/windows/apps/windows.media.mediaproperties.mediaencodingprofile.createmp3.aspx).
 
-- If a full path is not provided, the recording is placed in the AppData/temp directory. This can be accessed via the `File` API using `LocalFileSystem.TEMPORARY` or 'ms-appdata:///temp/<filename>' URI.
+- If a full path is not provided, the recording is placed in the AppData/temp directory. This can be accessed via the `File` API using `LocalFileSystem.TEMPORARY` or `ms-appdata:///temp/<filename>` URI.
```
Comment (rakatyal) [inline README.md]: `Appdata/temp`

File: README.md [hunk 2]
```diff
@@ -58,10 +58,6 @@ Although in the global scope, it is not available until after the `deviceready`
 
 - Only one media file can be played back at a time.
 
-- There are strict restrictions on how your application interacts with other media. See the [Microsoft documentation for details][url].
-
-[url]: http://msdn.microsoft.com/en-us/library/windowsphone/develop/hh184838(v=vs.92).aspx
-
```
Comment (dblotsky) [inline README.md]: Is this no longer relevant?

Comment (riknoll) [inline README.md]: The link redirects to a redirect to a page that does not sound like the original one that was linked to

Comment (dblotsky) [inline README.md]: Gotcha.

File: README.md (hunk 1 above)
Comment (riknoll) [inline README.md]: Fixed

**Comment by rakatyal** [comment]
LGTM.

**Comment by dblotsky** [comment]
Ell Gee Tee Emm.

**Pull request closed without merging**